test/dialect/mssql/test_reflection.py

```diff
@@ -319,6 +319,30 @@ def test_indexes_cols_with_spaces(self):
 
         eq_(set(list(t2.indexes)[0].columns), set([t2.c["x col"], t2.c.y]))
 
+    @testing.provide_metadata
+    def test_indexes_with_filtered(self):
```

Comment:
**mike bayer** (zzzeek) wrote:

there's a "connection" fixture you can use now:

    def test_indexes_with_filtered(self, conneciton):

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2142

Reply:
**Ramon Williams** (RamonWill) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2142